index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello-Microverse</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hello-Microverse</h1>
```

Comment:
- [ ] I would kindly suggest you give your `h1` tag a class name, so that the class name can be used to style the content in the tag
E.g 👇 

```HTML
<h1 class="heading">Hello-Microverse</h1>
```